rak811.go

```diff
@@ -97,12 +98,17 @@ func (l *Lora) HardReset() error {
 
 	// After the reset need to drain the response from the serial port.
 	reader := bufio.NewReader(l.port)
-	_, err = reader.ReadString('\n')
-	if err != nil {
-		return fmt.Errorf("read err:%v", err)
+	var resp string
+	for x := 0; x < 4; x++ {
```

Comment:
Do you know why the same operation on the python equivalent looks different? https://github.com/AmedeeBulle/pyrak811/blob/master/rak811/rak811.py#L187, especially the on the response handling side

Reply:
I think they are doing some flushing somewhere before trying to read any data again.
Will probably try to improve again, but for now this works well and stable.